HowTo.md

```diff
@@ -0,0 +1,27 @@
+### How to update favicon
+
+Favicons are the little icon next to the title of the page
+
+1. First download a png of the favicon you want
+2. Open the public folder and right click the images folder inside
+3. There should be an option that says Upload...
+4. Click that and upload your file
+5. Delete the old favicon file and rename your new one to "favicon.png"
+
+### How to make a new page
+
+1. Go to the menu.json file 
+2. add this to the end of the main area
+        {
+          "name": "Your Page",
+          "url": "/your-page"
+        }
```

Comment:
This is a great place to use code blocks so it renders neatly. We do this by using three `` ` `` (backtick) characters, like this:
````
```<programming language name for syntax highlighting, our code block language here is called "json">
<my code>
```
````

Here's the code block above as we would write it in this HowTo.md file:
````md
```json
{
  "name": "Your Page",
  "url": "/your-page"
}
```
````

Here's what this looks like when I use it in this comment:
```json
{
  "name": "Your Page",
  "url": "/your-page"
}
```